https://git.reviewboard.kde.org/r/124675/
Fix Bug 311991 - Taskbar buttons for minimized apps should not use disabled state
diff --git a/applets/taskmanager/package/contents/ui/Task.qml b/applets/taskmanager/package/contents/ui/Task.qml
index f655801ab1f7b1a9a915f35149c858f0ede22a29..78115e7fb617e1dda341e850d3f671f23d38b36c 100644
--- a/applets/taskmanager/package/contents/ui/Task.qml
+++ b/applets/taskmanager/package/contents/ui/Task.qml
@@ -250,7 +250,7 @@ MouseArea {
             visible: false
 
             active: task.containsMouse
-            enabled: !(model.Minimized && !plasmoid.configuration.showOnlyMinimized)
+            enabled: true
 
             source: model.DecorationRole
 
@@ -305,7 +305,7 @@ MouseArea {
 
         visible: (inPopup || !iconsOnly && !model.IsLauncher && (parent.width - LayoutManager.horizontalMargins()) >= (theme.mSize(theme.defaultFont).width * 5))
 
-        enabled: !model.Minimized
+        enabled: true
 
         text: (!inPopup && iconsOnly) ? "" : model.DisplayRole
         color: theme.textColor
